packages/grafana-data/src/types/dataFrameTypes.ts

```diff
@@ -0,0 +1,11 @@
+/**
+ * See also:
+ * https://github.com/grafana/grafana-plugin-sdk-go/blob/main/data/frame_type.go
+ *
+ * @public
+ */
+export enum DataFrameType {
```

Comment:
There's plenty i don't understand here, but this bit i have a smidge of knowledge - we don't want a value in the enum to represent an empty string, corresponding to the one that was added in the Go code?

Reply:
There is no value in an additional enum for "unknown" -- in go that is our alternative to using pointers and we have "omitempty" in the json settings.  We don't need to worry about "i know it is unknown vs i don't know"

Ideally we never get empty strings from the system, but if we do javascript does not really care -- most of our checks will be something like:
```
if (frame.meta?.type) {
...
}
```
and that will only be true if `meta.type` has a value with some length